packages/react-html/README.md

```diff
@@ -63,12 +63,12 @@ The children to be rendered inside the `#app` div.
 **styles**
 Descriptors for any style tags you want to include in the HEAD of the document.
 
-**synchronousScripts**
-Descriptors for any script tags you want to include in the HEAD of the document.
-
-**deferredScripts**
+**scripts**
 Descriptors for any deferred script tags you want to include at the end of the document.
```

Comment:
can we reword this? Maybe something like:

**scripts**
Descriptors for any script tags you want to include in your document. All scripts passed in here will be deferred by appending them to the end of the document. We encourage this as a default, although you may use `blockingScripts` for any scripts that **must** be included in the `HEAD` of the `document`.